common/djangoapps/student/models.py

```diff
@@ -2066,10 +2066,14 @@ def add_to_profile_url(self, course_key, course_name, cert_mode, cert_url, sourc
 
     def _cert_name(self, course_name, cert_mode):
         """Name of the certification, for display on LinkedIn. """
-        return self.MODE_TO_CERT_NAME.get(
```

Comment:
It's current location, a constant styled class variable on the class that this function is a member of seems to make sense to me. I'm not sure I would prefer a function scoped variable which contained the mapping dict.